yappi/_yappi.c

```diff
@@ -1911,7 +2037,7 @@ static PyMethodDef yappi_methods[] = {
     {"start", start, METH_VARARGS, NULL},
     {"stop", (PyCFunction)stop, METH_NOARGS, NULL},
     {"enum_func_stats", enum_func_stats, METH_VARARGS, NULL},
-    {"enum_thread_stats", enum_thread_stats, METH_VARARGS, NULL},

```

Comment:
The only thing I would say is to leave `enum_thread_stats` as it is and call `enum_context_stats` as it is probably used in lots of places and breaking BC on that might not be a good idea.

Reply:
Done!